doc/classes/SceneTree.xml

```diff
@@ -90,6 +90,7 @@
 			<return type="Tween" />
 			<description>
 				Creates and returns a new [Tween]. The Tween will start automatically on the next process frame or physics frame (depending on [enum Tween.TweenProcessMode]).
+				[b]Note:[/b] When creating a [Tween] using [method SceneTree.create_tween], the [Tween] will be detached from the actual [Node]. It will continue to animate even if the [Node] is freed, but it will automatically finish if there's nothing left to animate. If you want the [Tween] to be automatically killed when the [Node] is freed, prefer using [method Node.create_tween].
```

Comment:
```suggestion
				[b]Note:[/b] When creating a [Tween] using this method, the [Tween] will be detached from the actual [Node]. It will continue to animate even if the [Node] is freed, but it will automatically finish if there's nothing left to animate. If you want the [Tween] to be automatically killed when the [Node] is freed, prefer using [method Node.create_tween].
```
No need to mention the method itself here

Reply:
Thanks, mention removed